.github/CODEOWNERS

```diff
@@ -12,31 +12,33 @@
 # Virtual teams
 /x-pack/plugins/rule_registry/ @elastic/rac
 
-# App
-/x-pack/plugins/discover_enhanced/ @elastic/kibana-app
-/x-pack/plugins/lens/ @elastic/kibana-app
-/x-pack/plugins/graph/ @elastic/kibana-app
-/src/plugins/advanced_settings/ @elastic/kibana-app
-/src/plugins/charts/ @elastic/kibana-app
-/src/plugins/discover/ @elastic/kibana-app
-/src/plugins/management/ @elastic/kibana-app
-/src/plugins/kibana_legacy/ @elastic/kibana-app
-/src/plugins/timelion/ @elastic/kibana-app
-/src/plugins/vis_default_editor/ @elastic/kibana-app
-/src/plugins/vis_type_metric/ @elastic/kibana-app
-/src/plugins/vis_type_table/ @elastic/kibana-app
-/src/plugins/vis_type_tagcloud/ @elastic/kibana-app
-/src/plugins/vis_type_timelion/ @elastic/kibana-app
-/src/plugins/vis_type_timeseries/ @elastic/kibana-app
-/src/plugins/vis_type_vega/ @elastic/kibana-app
-/src/plugins/vis_types/vislib/ @elastic/kibana-app
-/src/plugins/vis_types/xy/ @elastic/kibana-app
-/src/plugins/vis_types/pie/ @elastic/kibana-app
-/src/plugins/visualize/ @elastic/kibana-app
-/src/plugins/visualizations/ @elastic/kibana-app
-/src/plugins/chart_expressions/expression_tagcloud/ @elastic/kibana-app
-/src/plugins/url_forwarding/ @elastic/kibana-app
-/packages/kbn-tinymath/ @elastic/kibana-app
+# Data Discovery
+/src/plugins/discover/ @elastic/kibana-data-discovery
+/x-pack/plugins/discover_enhanced/ @elastic/kibana-data-discovery
```

Comment:
would it make sense to also add our functionals?

Reply:
Added it.